DockerJenkins/README.rst

```diff
@@ -0,0 +1,95 @@
+
+==============================================
+Jenkins in docker used to render documentation
+==============================================
+
+.. default-role:: code
+.. highlight:: shell
+
+Prepare Jenkins in Docker
+-------------------------
+
+1. Follow all steps mention in [a Install Docker](../README.rst#prepare-docker)
+
+2. Copy all content of this directory (including all subdirectories) to a location on your docker host
+
+   (e.g. /opt/docker/jenkins)
+    
+3. Change directory to dockerfiles
+
+4. Build the Docker image
+
+   execute buildDockerImage.sh
+
+5. Create the Jenkins container
+
+   execute dockerRun.sh
+    
+   Please do a restart of the Jenkins container after you finished its setup
+    
+
+
+Basic understanding
+-------------------
+
+The main issue that should be addressed when running Jenkin in a Cocker container is
+that  without some preparation you are not able to launch other Docker containers.
+
+A `blog entry of Felix Heppner <https://www.oose.de/blogpost/jenkins-in-docker-und-mit-docker-und-fuer-docker>`__ 
```

Comment:
This is a blogpost in German. The blogpost refers to some English texts. This is my personal opinion: I do not like links to German texts or videos on an English documentation page. If possible, please try to find an English resource. Please think about people who do not master German. Other people may not be bothered by this, but I would try to avoid it unless absolutely necessary for some reason.

Reply:
You're right - I'll remove this section. Thank you